<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leerPendientes</class>
 <widget class="QDialog" name="leerPendientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leer</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Leer pendientes</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="idLeer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBuscar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>351</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID [1-999]</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descLeer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>239</width>
     <height>64</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Seleccione la ID y pulse &quot;Buscar&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Descripcion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="respLeer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>239</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Responsable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="estadoLeer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Limpiar campos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBuscar_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
